js/src/launchAsync.ts

```diff
@@ -203,7 +205,7 @@ export function launchAsync(token: string, subdomain: string, content: Content,
                         container: iframeContainer,
                         sessionId: response.sessionId,
                         charactersProcessed: response.meteredContentSize,
-                        postLaunchOperations: {
+                        postLaunchOperations: options.disablePlayPause ? null : {
```

Comment:
'providePlayPauseControl === true', or equivalent 'if (providePlayPauseControl)' to check truthy

Reply:
yes, you are going to change this to a true/false option - providePlayPauseControl is going to be true/false

Reply:
done